### DISCUSSION TITLE
beta2 is out

### DISCUSSION THREAD

**Discussion by filipkappa** [mdbootstrap/TW-Elements#1586] [Announcements]
[v1.0.0-beta2](https://github.com/mdbootstrap/tailwind-elements/releases/tag/v1.0.0-beta2) brings significant improvements to the optimization of JavaScript files. 

### Key optimizations
As part of this release, we have **[migrated from Webpack to Vite](https://tailwind-elements.com/docs/standard/getting-started/quick-start/#vite)** and **[added support for ES modules](https://tailwind-elements.com/docs/standard/getting-started/optimization/#modules)**. These changes have resulted in a significant reduction in the file size of components built with Tailwind Elements, which can greatly **improve the performance of your applications**.

### System dark theme on docs
Additionally, we have implemented a new **system option for a dark theme** in our documentation, which can make it easier to read in low-light environments.

### Integrations with frameworks
We've checked **[your votes & discussions](https://github.com/mdbootstrap/Tailwind-Elements/discussions)** regarding **integrations with external tools**, and we are confident that Tailwind Elements will work seamlessly with your existing workflows. 

We have **prepared dedicated guides** for:

- [Angular](https://tailwind-elements.com/docs/standard/integrations/angular-integration/)
- [Django](https://tailwind-elements.com/docs/standard/integrations/django-integration/)
- [Express](https://tailwind-elements.com/docs/standard/integrations/express-integration/)
- [Laravel](https://tailwind-elements.com/docs/standard/integrations/laravel-integration/)
- [Next](https://tailwind-elements.com/docs/standard/integrations/next-integration/)
- [Nuxt](https://tailwind-elements.com/docs/standard/integrations/nuxt-integration/)
- [React](https://tailwind-elements.com/docs/standard/integrations/react-integration/)
- [Remix](https://tailwind-elements.com/docs/standard/integrations/remix-integration/)
- [Solid](https://tailwind-elements.com/docs/standard/integrations/solid-integration/)
- [Svelte](https://tailwind-elements.com/docs/standard/integrations/svelte-integration/)
- [Vue](https://tailwind-elements.com/docs/standard/integrations/vue-integration/)

Overall, this release represents a major step forward for Tailwind Elements. 
We are excited to see the positive impact it will have on your projects!

Keep coding!

---

| ![Tailwind Elements logo](https://mdbootstrap.com/img/Marketing/general/logo/big/te.png) | 🏅 **[Contribute](https://tailwind-elements.com/docs/standard/getting-started/contribute/)** & help Tailwind Elements grow <br> 👁️ Track our **[live progress](https://github.com/orgs/mdbootstrap/projects/1)** on the next release <br>🗣️ **[Motivate us](https://github.com/mdbootstrap/Tailwind-Elements/discussions/categories/kind-words-general-feedback)** with some words of encouragement <br>💡 Share your **[ideas & feature requests](https://github.com/mdbootstrap/Tailwind-Elements/discussions/categories/share-ideas-request-features)** <br>💬 Engage, discuss & have fun in our [open-source community](https://github.com/mdbootstrap/Tailwind-Elements/discussions) |
|---|:---|

**Comment by HanseVlad** [comment]
Hello Tailwind Elements developers.
I'm using Tailwind Elements on Nuxt 3 and Tailwind

     "nuxt": "^3.3.3",
     "nuxt-delay-hydration": "^1.1.4",
     "postcss": "^8.4.21",
     "tailwindcss": "^3.3.1"

The new version 1.0.0-beta2 has some new bugs, namely:
in the documentation under Step 8 "Alternatively you can import Tailwind Elements by ..." is the file name 
src: "**https://cdn.jsdelivr.net/npm/tw-elements/dist/js/tw-elements.min.es.js**". 
wrong.
This new file is called
src: "**https://cdn.jsdelivr.net/npm/tw-elements/dist/js/tw-elements.es.min.js**".

When I try to correct the file's name, I become the error-message:
"**Uncaught SyntaxError**: export declarations may only appear at top level of a module tw-elements.es.min.js:11239"

 The initialization method from Step 6 works not correctly while 
the file "tw-elements.**min.es**.js" is now called "tw-elements.**es.min**.js" and cannot be found bei old name.

I hope you will fix these bugs quickly.

**Reply by juujisai** [reply]
Thanks for letting us know. With nuxt we suggest using npm, as CDN is primarily an option for testing and may encounter issues.

**Comment by garyross4** [comment]
Congrats!

**Comment by AgathaJarynski** [comment]
Kudos to @juujisai for introducing the JS modules!
